Project-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js

```diff
@@ -1,2 +1,17 @@
 import React from 'react';
+import CardBanner from './CardBanner';
+import CardContent from'./CardContent';
 import './Card.css';
+
+const CardContainer = () => {
+  return (
+    <div class="card-container">
```

Comment:
```suggestion
    <div className="card-container">
```
You need to use className in React or it will throw an error while still working. Since class is a reserved word in javascript.